hermes-consumers/src/test/groovy/pl/allegro/tech/hermes/consumers/consumer/filtering/FilterChainSpec.groovy

```diff
@@ -129,7 +129,7 @@ class FilterChainSpec extends Specification {
         def order
 
         RecordingSubscriptionMessageFilterCompiler(String type, AtomicInteger counter) {
-            super(type, { m -> test(m) })
+            super(type, null)
```

Comment:
Why did you change it to null?

Reply:
With newer version of groovy-spock lib you cannot pass method of instance which hasn't been constructed yet.

Reply:
so the 2nd argument wasn't used at all?

Reply:
Yes it wasn't, this overrided method was called because  RecordingSubscriptionMessageFilterCompiler extends MessageFilter